lib/models/article.js

```diff
@@ -498,4 +498,33 @@ Object.defineProperty(Article.prototype, 'isWeekend', {
 	}
 });
 
+Object.defineProperty(Article.prototype, 'contentClassification', {
+	get : function(){
+		if(this._contentClassification){
+			return this._contentClassification;
+		}
+
+		var results = /cms\/s\/([0-3])\//i.exec(this.raw.item.location.uri);
```

Comment:
need to be careful here. api is basically undocumented, so no idea if location is always there. 

safer to do `if (!this.raw.item.location) { return 0; }` above here imho.